clang/include/clang/StaticAnalyzer/Core/AnalyzerOptions.def

```diff
@@ -407,6 +407,11 @@ ANALYZER_OPTION(
 ANALYZER_OPTION(unsigned, MaxSymbolComplexity, "max-symbol-complexity",
                 "The maximum complexity of symbolic constraint.", 35)
 
+// HACK:https://discourse.llvm.org/t/rfc-make-istainted-and-complex-symbols-friends/79570
+// Ideally, we should get rid of this option soon.
+ANALYZER_OPTION(unsigned, MaxTaintedSymbolComplexity, "max-tainted-symbol-complexity",
+                "[DEPRECATED] The maximum complexity of symbolic to carry taint", 10)
```

Comment:
```suggestion
                "[DEPRECATED] The maximum complexity of a symbol to carry taint", 10)
```
Or something else might be even better, but the adjective "symbolic" is awkward without a noun.